src/main/java/com/rocketden/main/model/Room.java

```diff
@@ -21,4 +28,11 @@
     private String roomId;
 
     private LocalDateTime createdDateTime = LocalDateTime.now();
+
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "host_id")
```

Comment:
Maybe this can be a OneToOne relationship? That might simplify the use case, and also it might make sense that a user can't host two rooms at the same time.

Reply:
This makes sense, thanks for noting that -- done!